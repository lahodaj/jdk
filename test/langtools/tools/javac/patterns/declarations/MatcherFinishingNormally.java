/**
 * @test
 * @enablePreview
 * @compile/fail/ref=MatcherFinishingNormally.out -XDrawDiagnostics MatcherFinishingNormally.java
 */
public class MatcherFinishingNormally {
    public pattern MatcherFinishingNormally(String content) {}
}