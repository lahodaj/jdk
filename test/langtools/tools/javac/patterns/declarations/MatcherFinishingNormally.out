MatcherFinishingNormally.java:9:5: compiler.err.missing.match.stmt
- compiler.note.preview.filename: MatcherFinishingNormally.java, DEFAULT
- compiler.note.preview.recompile
1 error
